### PR TITLE
feat(AP-994): send client name in data payload

### PR DESCRIPTION
**Pull request by mattstrom**

### CODE REVIEW

File: src/beacon.js [hunk 1]
```diff
@@ -67,6 +90,7 @@ export default function Emitter(endpoint, context, blockTransmit) {
         let editedMessage = message;
         editedMessage = message.payload || {};
         editedMessage.type = message.type;
+        editedMessage.client = opts.clientName;
```
Comment (rmamchyk) [inline src/beacon.js:94]: Do we need to send the client type to the `/events` endpoint as well?

Comment (mattstrom) [reply src/beacon.js:94]: I wondered about this too. I'll check with @FrazerBayley.

File: src/beacon.js [hunk 2]
```diff
@@ -1,4 +1,5 @@
 import retrieve from './retrieve.js';
+import { assign, omitUndefined } from "./ponyfills/objects.js";
```
Comment (rmamchyk) [inline src/beacon.js:2]:
```suggestion
import { assign, omitUndefined } from './ponyfills/objects.js';
```

File: src/beacon.js (hunk 1 above)
Comment (mattstrom) [reply src/beacon.js]: I verified with Frazer that the `/events` endpoint does not need the `client` value.

**Review by rmamchyk — APPROVED** [review]

**Pull request merged**